### ISSUE TITLE
Unable to specify a port for Remote run configuration

### ISSUE THREAD

**Issue by dtsaryov** [JetBrains/gradle-idea-ext-plugin#27]
Tested for all versions (0.1, 0.2, 0.3)

Desirable configuration:
```Groovy
idea.project.settings.runConfigurations {
    "localhost:8080"(Remote) {
        host 'localhost'
        port 8787
    }
}
```

But the default port is specified in created configuration:

![image](https://user-images.githubusercontent.com/6183810/43415163-e57f02ac-9445-11e8-9a66-84beaddd72f0.png)

The issue can be reproduced with this small project:

[ideaplugin.zip](https://github.com/JetBrains/gradle-idea-ext-plugin/files/2242367/ideaplugin.zip)

**Comment by nskvortsov** [comment]
A bug inside corresponding IDEA code.

**Comment by nskvortsov** [comment]
logged as https://youtrack.jetbrains.com/issue/IDEA-196489

**Comment by nskvortsov** [comment]
Fixed in IDEA, will be shipped with 2018.2.x
